setup.sh

```diff
@@ -117,28 +117,33 @@ function reset() {
     echo "----------------------------"
     echo "Reseting branch and virtual env"
     echo "----------------------------"
+
     if [ "1" == $(git branch -vv |grep -cE "\* develop|\* master") ]
     then
-        if [ -d ".env" ]; then
-          echo "- Delete your previous virtual env"
-          rm -rf .env
-        fi
 
-        git fetch -a
+        read -p "Reset git branch? (This will remove all changes you made!) [y/N]? "
+        if [[ $REPLY =~ ^[Yy]$ ]]; then
 
-        if [ "1" == $(git branch -vv |grep -c "* develop") ]
-        then
-          echo "- Hard resetting of 'develop' branch."
-          git reset --hard origin/develop
-        elif [ "1" == $(git branch -vv |grep -c "* master") ]
-        then
-          echo "- Hard resetting of 'master' branch."
-          git reset --hard origin/master
+            git fetch -a
```

Comment:
Confirmation for git reset is really nice.

But should not `git fetch` be done before?

Reply:
yes, but for now the script is still required - so i would rather not have 20 issues with it failing until it's gone.

not sure (not sure there is a difference) ...

if we don't reset it doesn't matter if fetch was executed ... (we don't check if changes did happen - the script just "blindly" reset).
If we reset, we fetch and use the latest version ...

Reply:
okay, that was not a change request...